VideoWeb/VideoWeb/ClientApp/src/app/vh-officer/unread-messages/unread-messages.component.ts

```diff
@@ -82,6 +82,20 @@ export class UnreadMessagesComponent extends UnreadMessagesComponentBase impleme
     handleImReceived(message: InstantMessage) {
         if (this.getHearing().id === message.conferenceId) {
             this.incrementUnreadCounter(message.conferenceId, message.from);
+
+            const messagesPendingToRead = this.unreadMessages.find(x => x.participant_id === message.from);
+            if (messagesPendingToRead) {
```

Comment:
could we add a unit test for this, to check if the number of unread messages is included in the debug message?

Reply:
I think there is no need to include a unit test in here. The var will contain the unread messages coming from a participant excluding the vho senders.. that explains the if/else... sometimes we need to log that info, sometimes we dont